TL improvement (#558)
07th-mod/higurashi-console-arcs: Update/omot_012.txt

//r…今のレナの格好で家まで帰れといわれたら、俺だったら旅に出る。それも海外に。いや太陽系脱出かも。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…今のレナの格好で家まで帰れといわれたら、俺だったら旅に出る。それも海外に。いや太陽系脱出かも。",

> 			NULL, "...If I had to walk home in the clothes Rena was wearing now, I'd go on a journey. Perhaps overseas. No, maybe even beyond the edge of the solar system.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...If I had to walk home in the clothes Rena was wearing now, I'd set off on a journey. Perhaps overseas. No, maybe even beyond the edge of the solar system.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "一体誰が考えたんだ、この罰ゲームは？！　クラスの男女交際事情を考えてみても、他の奴が引いたら確実にスルーされるセーフカードじゃないか！　明らかに俺と魅音を狙い撃ちしたとしか思えんぞ！！",

> 			NULL, "Who on earth thought up this punishment?! Just considering the gender relationships in the class, this is a safe card for everyone else because they'd surely disregard it even if they were to draw it! It was obviously created to set on Mion and me!!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Who on earth thought up this punishment?! Just considering the gender relationships in the class, this is a safe card for everyone else because they'd definitely disregard it even if they drew it! It was obviously created specifically for Mion and me!!", Line_Normal);
```
